Translation.Rmd

```diff
@@ -401,7 +405,7 @@ Before we begin, let's quickly cover how formulas are expressed in LaTeX. The fu
 * Most simple mathematical equations are written in the same way you'd type
   them in R: `x * y`, `z ^ 5`. Subscripts are written using `_` (e.g., `x_1`).
 
-* Special characters start with a `\`: `\pi` = π, `\pm` = ±, and so on.
+* Special characters start with a `\`: `\pi` = $\pi$, `\pm` = $\pm$, and so on.
```

Comment:
In my pdf, the pi symbol was not printing. Using LaTeX notation should be safer, I think.

Reply:
Good call